ExeterTeam25/group-software-project
# Report comments

What I've done: 
- Allowed for the reporting of comments 
- Added how to report things to Help page 

To test: 
- Make sure a post has comments 
- Click on "see comments" 
- Click on "report comment" 
- View and action reported comments as a game master
